term/fhir-term/src/main/java/com/ibm/fhir/term/service/FHIRTermService.java

```diff
@@ -936,8 +977,25 @@ public ValidationOutcome validateCode(ValueSet valueSet, Coding coding, Validati
             throw new UnsupportedOperationException("Validation parameters are not supported");
         }
         boolean result = ValueSetSupport.validateCode(valueSet, coding);
-        LookupOutcome outcome = result ? lookup(coding) : null;
-        return validateCode(coding, result, outcome);
+        if (result) {
+            LookupOutcome outcome = result ? lookup(coding) : null;
```

Comment:
result is always true there is dead code here.

Reply:
good call

Reply:
done